server/rating.js

```diff
@@ -4,46 +4,57 @@ import fetch from "node-fetch";
 export const displayRating = express.Router();
 
 async function getRatingData(id) {
-  const response = await fetch(
-    "https://plankton-app-xhkom.ondigitalocean.app/api/reviews?filters%5Bmovie%5D=" +
-      id
-  );
-  const reviewData = await response.json();
-  const data = reviewData.data;
-  return data;
+    const response = await fetch(
```

Comment:
Many of these lines have only changed with regards to indentation, which makes the actually meaningful changes harder to identify in a `git diff` like this.

I recommend using an auto-formatter, and when working in a team, make sure everyone use the same settings.